test/modules/CloudToDeviceMessageTester/CloudToDeviceMessageTester.csproj

```diff
@@ -32,6 +32,14 @@
     <Content Include="docker*/**/*.*" CopyToPublishDirectory="Always" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
why need these?

Reply:
These are automatically added when I added the files. 

And I see other projects that have Dockerfiles also have these lines

Reply:
Which project has these lines? I can't find it in TRC project. Try to remove it, see if it causes any problem.  And I think the project with these line doesn't show docker folder under that project.

Reply:
Removed it. Seems to be fine